test/parallel/test-dgram-empty-packet.js

```diff
@@ -1,22 +1,22 @@
 'use strict';
-var common = require('../common');
+const common = require('../common');
+const dgram = require('dgram');
 
-var dgram = require('dgram');
-var callbacks = 0;
-var client;
-var timer;
+let callbacks = 0;
+let timer;
 
 if (common.isOSX) {
   common.skip('because of 17894467 Apple bug');
   return;
 }
 
-client = dgram.createSocket('udp4');
+const client = dgram.createSocket('udp4');
+
+client.bind(0, function() { 
```

Comment:
`make test` or (`vcbuild test nosign` on Windows) would run the linter and would have caught this. If you want to run just the linter without running the whole test suite first, you can run the linter alone with `make jslint` (or, I think, `vcbuild jslint` on Windows).